src/Core/Module/ModuleManager.php

```diff
@@ -292,6 +292,66 @@ public function reset(string $name, bool $keepData = false): bool
         return $reset;
     }
 
+    /**
+     * Register a module on given hook.
+     *
+     * @param string $name The module name
+     * @param array $hooksName The hooks name
+     *
+     * @return bool True for success
+     */
+    public function registerHook($name, $hooksName)
```

Comment:
```suggestion
    public function registerHook(string $moduleName, array $hookNames)
```
I know the other methods use different naming, but it still deserves improvement No nee need to change it everywhere but we can at elast improve it on these new methods